### PR TITLE
Update system log tables TTL

### PR DESCRIPTION
**Pull request by srikanthccv**
high volume tables - 7 days
moderate volume tables - 15 days
low volume tables - 30 days

### CODE REVIEW

File: charts/clickhouse/templates/clickhouse-operator/configmaps/etc-configd-files.yaml [hunk 1]
```diff
@@ -36,7 +36,6 @@ data:
             <engine>Engine = MergeTree PARTITION BY event_date ORDER BY event_time TTL event_date + interval {{ .Values.clickhouseOperator.queryLog.ttl }} day</engine>
             <flush_interval_milliseconds>{{ .Values.clickhouseOperator.queryLog.flushInterval }}</flush_interval_milliseconds>
         </query_log>
-        <query_thread_log remove="1"/>
```
Comment (prashant-shahi) [inline charts/clickhouse/templates/clickhouse-operator/configmaps/etc-configd-files.yaml:39]: is this change intended?

Comment (srikanthccv) [reply charts/clickhouse/templates/clickhouse-operator/configmaps/etc-configd-files.yaml:39]: Yes, I enabled the `query_thread_log` with 7 days to better understand how many threads are used for each kind of operation. If 7 days is too much then we will reduce it further.
